pkg/cloud/services/networking/network_test.go

```diff
@@ -423,6 +424,260 @@ func Test_ReconcileExternalNetwork(t *testing.T) {
 	}
 }
 
+func Test_ReconcileSubnet(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	clusterName := "test-cluster"
+	expectedSubnetName := getSubnetName(clusterName)
+	expectedSubnetDesc := names.GetDescription(clusterName)
+	fakeSubnetID := "d08803fc-2fa5-4179-b9d7-8c43d0af2fe6"
+	fakeCIDR := "10.0.0.0/24"
+	fakeNetworkID := "d08803fc-2fa5-4279-b9f7-8c45d0ff2fe6"
+	fakeDNS := "10.0.10.200"
+
+	tests := []struct {
+		name             string
+		openStackCluster *infrav1.OpenStackCluster
+		expect           func(m *mock.MockNetworkClientMockRecorder)
+		want             *infrav1.OpenStackCluster
```

Comment:
The test doesn't check `want`, and if you add a check it fails.

Reply:
Fixed!